source/operating-a-service/incident-playbook.html.md

```diff
@@ -0,0 +1,91 @@
+# Incident playbook
+
+## Once an incident happens
+
+### 1. Form an incident response team
+
+Self-organise to appoint:
+
+1.  the **Incident comms lead**
+    (responsible for communicating and alerts to users and stakeholders)
+2.  the **Incident tech lead**
+    (responsible for technical direction and communicating to the comms lead)
+3. the **Incident support lead**, when the incident involves a public-facing service
```

Comment:
```suggestion
3. the **Incident support lead**
```

I think probably useful in all cases? & one less thing to consider when kicking off incident